programs/move_loader_api/src/lib.rs

```diff
@@ -147,30 +181,27 @@ pub fn process_instruction(
 
                 // TODO: Return errors instead of panicking
 
-                let args: Vec<TransactionArgument> = bincode::deserialize(&data).unwrap();
+                let invoke_info: InvokeInfo = bincode::deserialize(&data).unwrap();
```

Comment:
I don't love this.  Can you experiment with using `keyed_accounts[0].signer_key()` instead?

Reply:
Yup, the sender/signer stuff is WIP